src/App.tsx

```diff
@@ -12,7 +12,7 @@ import { setToken } from 'store/slices/authSlice';
 import './App.css';
 import Footer from 'components/Footer';
 import { createGlobalStyle } from 'styled-components';
-import BoardPage from 'pages/BoardPage';
+import BoardPage from 'pages/BoardPageCopy';
```

Comment:
We should rename BoardPageCopy to only BoardPage.

Reply:
Yep